js/index.d.ts

```diff
@@ -12,34 +12,34 @@ export namespace api {
         session?: (number|null);
 
         /** Command openChan */
-        openChan?: (api.IOpenChannel|null);
+        openChan?: (api.OpenChannel|null);
```

Comment:
These are still instances not interfaces.

Reply:
oops, due to an attempt to avoid converting non-interface types (enums), I accidentally also reverted this due to the types not being fully qualified u_u

it's now fixed!